config.py

```diff
@@ -6,72 +6,71 @@ class Config:
     def __init__(self):
         """Initializes the config class.
         """
-        # opens the config if it exists
-        if os.path.exists(os.path.join('resources', 'config.json')):
-            with open(os.path.join('resources', 'config.json')) as config:
+        config_path = os.path.join('resources', 'config.json')
+        # open the config if it exists
+        if os.path.isfile(config_path):
+            with open(config_path) as config:
                 self.config = json.load(config)
         else:
             # if config doesn't exist, create it and write brackets so it can be loaded by the json module
-            with open(os.path.join('resources', 'config.json'), "w") as config:
-                config.write(config := "{}")
+            with open(config_path, 'w') as config:
+                config.write(config := '{}')
                 self.config = json.loads(config)
 
+        resourcepath = lambda x: os.path.join('resources', x)
+        fileexists = lambda x: os.path.isfile(resourcepath(x))
         if self.read_keys() is None:
-            if os.path.exists(os.path.join('resources', 'unfixed-info.bin')) and os.path.exists(
-                    os.path.join('resources', 'locked-secret.bin')):
-                self.write_key_path(
-                    (os.path.join('resources', 'unfixed-info.bin'), os.path.join('resources', 'locked-secret.bin')))
-            elif os.path.exists(os.path.join('resources', 'key_retail.bin')):
-                self.write_key_path((os.path.join('resources', 'key_retail.bin'), ))
+            if fileexists('unfixed-info.bin') and fileexists('locked-secret.bin'):
+                self.write_key_paths(resourcepath('unfixed-info.bin'), resourcepath('locked-secret.bin'))
+            elif fileexists('key_retail.bin'):
+                self.write_key_paths(resourcepath('key_retail.bin'))
 
         if self.get_region_path() is None:
-            if os.path.exists(os.path.join('resources', 'regions.json')):
-                self.write_region_path(os.path.join('resources', 'regions.json'))
-            elif os.path.exists(os.path.join('resources', 'regions.txt')):
-                self.write_region_path(os.path.join('resources', 'regions.txt'))
+            if fileexists('regions.json'):
+                self.write_region_path(resourcepath('regions.json'))
+            elif fileexists('regions.txt'):
+                self.write_region_path(resourcepath('regions.txt'))
 
         if self.get_update_status() is None:
             self.set_update(True)
 
         if self.get_color() is None:
             self.write_color('DarkBlue3')
 
-    def write_key_path(self, key_path: tuple):
+    def write_key_paths(self, *key_paths):
```

Comment:
Function name was changed from write_key_path => write_key_paths. This causes an error as it is still referred to write_key_path when it gets called in main.py.

Reply:
Sorry about this, I was kind of tired when I made this PR. Would you like me to fix the error, or revert the name to the old one? I think the new name makes more sense if the function is meant to write a variable number of paths.

Reply:
i'd say fixing the issue is fine, if you need to know where it's referenced in main.py lmk